README.md

```diff
@@ -121,16 +136,16 @@ Below is a comprehensive list showing all zbox commands along with their respect
-| get-diff          | [Get differences](#get-differences): Get differences between the local files specified by `localpath` and the files stored on the root remotepath of the allocation.<br /><br /><br />[Get differences with excludepath](#get-differences): Get differences between local directory and root remotepath and exclude a specific remotepath. | Get differences:`./zbox get-diff --allocation $ALLOCATION_ID --localpath <path_to_local_directory>`<br /><br />Get differences with excludepath:`./zbox get-diff --allocation $ALLOCATION_ID --localpath /home/zus/files --excludepath /myfiles` |
+| sync              | [Sync](#sync): Sync all files from the local directory to root path on an allocation.<br /><br />[Sync to specifed path](#sync): Sync all files from the local directory to specified path on an allocation.<br /><br />[Batch Upload files using Sync](#sync): Upload multiple files at once from a local directory. <br /><br />[Sync with Excludepath](#sync): Exclude specific directories on an allocation during sync.<br /><br />[Sync with chunknumber](#sync): Sync with chunk number to control the amount of data send in one http multipart request to blobbers.By default its set to 1 which will only send 64KB of data per request. | Sync:`./zbox sync --localpath /home/zus/files --allocation $ALLOCATION_ID`<br /><br />Sync to specifed path:`./zbox sync --localpath /home/zus/files --remotepath /myfiles --allocation $ALLOCATION_ID`<br /><br />Batch Upload Files using Sync: `./zbox sync --uploadonly --localpath /home/zus/files --remotepath /myfiles `<br /><br /><br />Sync with Excludepath: `./zbox sync --allocation $ALLOCATION_ID --localpath /home/zus/files --remotepath /myfiles --excludepath /myfiles/audio.mp3 `<br /><br /><br />Sync with chunknumber:`zbox sync --allocation $alloc --localpath /home/zus/files --remotepath /myfiles --chunknumber 100` |
```

Comment:
Batch Upload file Command is not user friendly its param should be Batch or --batchupload instead of uploadonly it does not makes any sense

Reply:
fix the vertical spacing of all commands with its usages to make it look good

Reply:
the param changes has to be made from the code only ,i cannot do this, i can only document the given param.

Reply:
For vertical spacing and alignment see my comment at top.